gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dsubmenubutton_ttt2.lua

```diff
@@ -69,9 +70,11 @@ end
 
 ---
 -- @param Material iconMat
+-- @param boolean iconFullSize
 -- @realm client
-function PANEL:SetIcon(iconMat)
+function PANEL:SetIcon(iconMat, iconFullSize)
 	self.contents.icon = iconMat
+	self.contents.iconFullSize = iconFullSize and true or false
```

Comment:
```suggestion
	self.contents.iconFullSize = iconFullSize
```
😆 

If you wanna make sure it's never `nil`, you could cut out the `and true`.

Reply:
nah, nil is not the problem, I wanted to make sure its always a bool, for some reason, that someone inputs a material for example. So I guess tobool() would suffice. But yeah as I only check it and dont use it, just leave it away :stuck_out_tongue_closed_eyes: